<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>348</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Commodore 64 emulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QWidget" name="containerWidget" native="true">
      <zorder>toolBox</zorder>
     </widget>
    </item>
    <item>
     <widget class="QToolBox" name="toolBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>297</width>
         <height>276</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Memory</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QSpinBox" name="hexBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
            <property name="displayIntegerBase">
             <number>16</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="hexValue">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="maxLength">
             <number>4</number>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>309</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuROM">
    <property name="title">
     <string>ROM</string>
    </property>
    <widget class="QMenu" name="menuLoad">
     <property name="title">
      <string>Recent</string>
     </property>
    </widget>
    <addaction name="actionLoad_2"/>
    <addaction name="actionStart"/>
    <addaction name="menuLoad"/>
   </widget>
   <widget class="QMenu" name="menuFont">
    <property name="title">
     <string>Font</string>
    </property>
    <addaction name="actionChange"/>
   </widget>
   <widget class="QMenu" name="menuCode">
    <property name="title">
     <string>Code</string>
    </property>
    <widget class="QMenu" name="menuRecent_files">
     <property name="title">
      <string>Recent files</string>
     </property>
    </widget>
    <addaction name="actionLoadCode"/>
    <addaction name="menuRecent_files"/>
   </widget>
   <addaction name="menuROM"/>
   <addaction name="menuFont"/>
   <addaction name="menuCode"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionLoad_2">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionChange">
   <property name="text">
    <string>Change...</string>
   </property>
  </action>
  <action name="actionLoadCode">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
